internal/tfengine/tfengine.go

```diff
@@ -178,6 +179,14 @@ func dumpTemplate(conf *Config, pwd, cacheDir, outputPath string, ti *templateIn
 		return err
 	}
 
+	// Pass through specified keys that should be validated against the
+	// schema.
+	for _, k := range ti.Passthrough {
+		if v, ok := conf.Data[k]; ok {
```

Comment:
should we throw an error if the specified key in passthrough is invalid?

Reply:
Yes, that's probably a good idea. If it's being passthrough'd, then it's probably necessary.